scripts/scrape.py

```diff
@@ -29,6 +29,8 @@
 parser.add_argument('--neo4j-server', type=str, default='localhost',
                     help='The FQDN to the Neo4j server')
 parser.add_argument('--kerberos', action='store_true', help='Use Kerberos for authentication')
+parser.add_argument('--until', type=str,
```

Comment:
Optional: Putting this under the `--since` argument would be nice for readability.